python/whylogs/api/logger/experimental/logger/actor/process_rolling_logger_messages.py

```diff
@@ -180,7 +194,9 @@ class PubSubEmbeddingDict(TypedDict):
     log_embedding_request: LogEmbeddingRequestDict
 
 
-def _decode_pubsub_data(data: Union[PubSubEmbeddingDict, PubSubDict]) -> Optional[bytes]:
+def _decode_pubsub_data(
+    data: Union[PubSubEmbeddingDict, PubSubDict],
+) -> Optional[bytes]:
```

Comment:
This didn't actually change right? Looks like it shouldn't have been reformatted either based on the length.

Reply:
I think this might be something with whylogs' auto format tools? let me run them again

Reply:
PEP 8 suggests lines should be limited to 79 characters by default, and this line had 90 without the indents proposed by `flake8` here. however, our pre-commit hooks suggests it can go up to 160, so I'm not sure what might have triggered it. I can revert.